contrib/docker-compose/standalone/docker-compose.yml

```diff
@@ -58,6 +59,16 @@ services:
         max-size: "10m"
         max-file: "3"
 
+  haproxy-internal:
+    image: haproxy:2.6-alpine
+    restart: on-failure
```

Comment:
on-failure rather than always? should we make this consistent across all containers?

Reply:
👍🏻 Changed it to `always`